src/app/pages/fake-journal/__mocks__/fake-journal.mock.ts

```diff
@@ -5,235 +5,242 @@ const todayAt8Am = moment().startOf('day').add(8, 'hour');
+          title: 'Captain',
         },
-        driverNumber: 'PEARS015220A99HC',
-        mobileTelephone: '07654 123456',
+        driverNumber: 'CRAIG375220A99HC',
         primaryTelephone: '01234 567890',
-        secondaryTelephone: '04321 098765',
-        emailAddress: 'florence@pearson.com',
+        emailAddress: 'captain@jeremy.com',
       },
-      previousCancellation: [
-        'Act of nature',
-      ],
     },
     slotDetail: {
       duration: 57,
       slotId: `${end2endPracticeSlotId}_0`,
-      start: todayAt8Am.add(10, 'minutes').format(),
+      start: todayAt8Am.add(58, 'minutes').format(),
     },
     testCentre: {
       centreId: 54321,
       centreName: 'Example Test Centre',
       costCode: 'EXTC1',
     },
-    vehicleSlotType: 'Voc90mins',
+    vehicleSlotType: 'B57mins',
+    vehicleSlotTypeCode: 7,
   },
   {
     booking: {
       application: {
-        applicationId: 1234569,
+        applicationId: 1234572,
         bookingSequence: 1,
-        checkDigit: 9,
+        checkDigit: 3,
         entitlementCheck: false,
-        categoryEntitlementCheck: false,
+        specialNeedsCode: 'NONE',
+        specialNeedsExtendedTest: false,
         extendedTest: false,
         progressiveAccess: false,
-        testCategory: 'B',
-        vehicleGearbox: 'Automatic',
-        welshTest: true,
-        meetingPlace: 'Next to test centre',
-        specialNeedsExtendedTest: false,
-        specialNeedsCode: 'NONE',
+        testCategory: 'C',
+        vehicleGearbox: 'Manual',
+        vehicleHeight: 5,
+        vehicleLength: 15,
+        vehicleSeats: 50,
+        vehicleWidth: 2,
+        welshTest: false,
       },
       candidate: {
         candidateAddress: {
-          addressLine1: 'My House',
-          addressLine2: 'Someplace',
-          addressLine3: 'Sometown',
-          postcode: 'AB45 6CD',
+          addressLine1: 'The Gables Cottage',
+          addressLine2: 'Home Farm',
+          addressLine3: 'Farm Road',
+          addressLine4: 'Farm Area',
+          addressLine5: 'Farmtown',
+          postcode: 'FA43 9XY',
         },
-        candidateId: 103,
+        candidateId: 106,
         candidateName: {
-          firstName: 'Jane',
-          lastName: 'Doe',
-          title: 'Mrs',
+          firstName: 'James',
+          lastName: 'Brown',
+          title: 'Mr',
         },
-        driverNumber: 'DOEXX625220A99HC',
+        driverNumber: 'BROWN915220A99HC',
+        gender: 'M',
         mobileTelephone: '07654 123456',
-        emailAddress: 'jane@doe.com',
+        primaryTelephone: '01234 567890',
+        secondaryTelephone: '04321 098765',
+        dateOfBirth: '1950-02-27',
+        ethnicityCode: 'A',
       },
-      previousCancellation: [
-        'DSA',
-        'Act of nature',
-      ],
     },
     slotDetail: {
       duration: 57,
       slotId: `${end2endPracticeSlotId}_1`,
-      start: todayAt8Am.add(57, 'minutes').format(),
+      start: '2022-01-20T13:35:00',
```

Comment:
We need to make a dynamic date so we are always viewing todays journal, an example of this would be the line above. Please do this for all slots and make them different times please